apps/member-profile/app/routes/_profile.companies_.$id.tsx

```diff
@@ -4,6 +4,7 @@ import {
   type SerializeFrom,
 } from '@remix-run/node';
 import { generatePath, Link, useLoaderData } from '@remix-run/react';
+import { useSearchParams } from '@remix-run/react';
```

Comment:
It seems there is also `useSearchParams` from `@oyster/ui` [here](https://github.com/tomas-salgado/oyster/blob/d34f0dc050bc819a52583a0338bc1148baf256be/packages/ui/src/hooks/use-search-params.ts). Noting to investigate if I should use this instead